src/fetch.rs

```diff
@@ -102,25 +109,22 @@ mod cert_authority {
     use openssl::x509::X509;
     use reqwest::StatusCode;
 
-    #[derive(StructOpt)]
+    #[derive(Parser)]
     pub struct Args {
-        #[structopt(help = "Specify encoding to use for certificates. [PEM | DER]")]
+        /// Specify encoding to use for certificates.
+        #[arg(value_name = "encoding", required = true)]
         pub encoding: CertFormat,
 
-        #[structopt(
-            help = "Specify the processor model for the certificate chain. [Milan | Genoa]"
-        )]
+        /// Specify the processor model for the certificate chain.
+        #[arg(value_name = "processor-model", required = true)]
         pub processor_model: ProcType,
 
-        #[structopt(help = "Directory to store the certificates in.")]
+        /// Directory to store the certificates in.
+        #[arg(value_name = "certs-dir", required = true)]
         pub certs_dir: PathBuf,
 
-        #[structopt(
-            long,
-            short,
-            default_value = "vcek",
-            help = "Specify to pull the VLEK certificate chain instead of VCEK."
-        )]
+        ///  Specify to pull the VLEK certificate chain instead of VCEK.
```

Comment:
```suggestion
        ///  Specify what endorsement certificate chain to pull for: vcek or vlek. Defaults to vcek
```

If I understand the flag correctly, you have to use it and then point to the endorser you want, right?

So it would be --endorser vlek?

It wouldn't change the value by just passing the flag.